src/workerd/io/worker.h

```diff
@@ -422,6 +425,11 @@ class Worker::Api {
   //   that this doesn't have to allocate and so it's not possible to hold a lock while returning
   //   to the event loop.
 
+  virtual kj::Own<jsg::Lock> lock(jsg::V8StackScope& stackScope, const void* embedderData) const {
```

Comment:
Can we remove the other overload of `lock()` and update all call sites to use this one?

Otherwise it seems confusing if "sometimes" the embedder data is left null, just because some code wasn't updated. All the call sites should be easy to find.

Reply:
Once the internal repo is updated yes.